### PR TITLE
Style updates

### PR DESCRIPTION
**Pull request by sccolbert**
Initial set of tweaks to application styles. Covers sidebars, background, file browser, and command palette. edit: now covers tabs/panels too

capture
capture

### CODE REVIEW

**Comment by sccolbert** [comment]
@blink1073 @cameronoelsen

File: src/default-theme/filebrowser.css [hunk 1]
```diff
@@ -131,6 +141,12 @@
 }
 
 
+/* increate sepcificity to override bundled default */
```
Comment (blink1073) [inline src/default-theme/filebrowser.css]: increase specificity*

**Comment by sccolbert** [comment]
capture

**Comment by sccolbert** [comment]
This is ready to go. 

Things for a future PR:
- better styling of matched characters in command palette
- drop shadows for dock panels? (I personally prefer it without).
- spacing between dock panels (currently hard-coded in phosphide)

**Comment by sccolbert** [comment]
@ellisonbg fyi

**Comment by sccolbert** [comment]
@jasongrout fyi

File: src/default-theme/filebrowser.css [hunk 2]
```diff
@@ -131,6 +141,12 @@
 }
 
 
+/* increate specificity to override bundled default */
```
Comment (blink1073) [inline src/default-theme/filebrowser.css]: increase*

**Comment by blink1073** [comment]
LGTM.

**Comment by cameronoelsen** [comment]
@sccolbert @blink1073 Not sure who is updating the cells for JupyterLab, but here is the close button for the tabs. I included a 50x50 png version along with the SVG version. Is there a specific size the close icon should be?

tab 1 close button
[Close Icon.zip](https://github.com/jupyter/jupyter-js-plugins/files/134956/Close.Icon.zip)

**Comment by sccolbert** [comment]
@cameronoelsen We are currently using font awesome. But we can use that icon and make it any size you want.

**Comment by sccolbert** [comment]
Font awesome is easy to use because we don't need to serve a resources file alongside the app. Just have to load the font.

**Pull request merged**